python/google/protobuf/json_format.py

```diff
@@ -116,6 +117,28 @@ class ParseError(Error):
   """Thrown in case of parsing error."""
 
 
+class _UnknownEnumStringValueParseError(ParseError):
```

Comment:
Does this class inherit from `ParseError` (it looks like it does)? So we don't break code that tries to catch `ParseError`

Reply:
Correct. Added a comment about that.

Although I think the implementation will never leak `_UnknownEnumStringValueParseError`, it still gets renamed to `ParseError` in `_ConvertFieldValuePair`.. but I wouldn't rely on that, the code is very hard to follow :) It's safer if I just inherit and then I know I didn't change any public interface